data/model/namespacequota.py

```diff
@@ -0,0 +1,371 @@
+import json
+import humanfriendly
+
+from peewee import fn, JOIN
+
+from data.database import (
+    Repository,
+    QuotaLimits,
+    UserOrganizationQuota,
+    QuotaType,
+    Manifest,
+    get_epoch_timestamp_ms,
+    Tag,
+    RepositorySize,
+)
+from data.model import (
+    InvalidOrganizationException,
+    _basequery,
+    organization,
+    user,
+    InvalidUsernameException,
+    repository,
+    notification,
+)
+
+HUMANIZED_QUOTA_UNITS = [i.decimal.symbol for i in humanfriendly.disk_size_units] + ["bytes"]
```

Comment:
I think the any sizes should be sent as standard bytes from the api, across all endpoints. Any conversion can be done by the clients (the UI in this case).

Reply:
Hmm, the reason I've added here is because it was easier to handle conversions in Python. I was not able to find a good library in angular that would convert bytes into human readable format. And since our UI will shift later, our conversion logic will remain intact in the backend. The `humanfriendly` library converts bytes to human readable format and the other way as well.

Reply:
Do we have examples of how this is done elsewhere?

Reply:
Hmm, examples in our codebase or what others did? I'd need to dig for that. I just din’t want to shift any logic to the UI. In my opinion UI should only display what backend sends.

Reply:
In our code base. I do like the idea of a uniform unit within the backend code at least up until the api sends the data (allowing the API to change the unit could be a happy medium) to the UI. However I would also like to take direction from what has been done previously.

Reply:
We usually return bytes when size is concerned,

Reply:
is this worth fixing right now or can we leave it for work as we fix issues from QE?

Reply:
We should fix it.